onnxscript/function_libs/torch_lib/ops/special.py

```diff
@@ -106,7 +106,7 @@ def aten_special_erfc(self: TFloatOrBFloat16) -> TFloatOrBFloat16:
 def aten_special_erfcx(self: TFloatOrBFloat16) -> TFloatOrBFloat16:
     """special_erfcx(Tensor self) -> Tensor"""
 
-    return op.Mul(op.Exp(op.Pow(self, 2)), op.Erf(self))
+    return op.Mul(op.Exp(op.Pow(self, 2)), op.Sub(1, op.Erf(self)))
```

Comment:
Maybe https://math.stackexchange.com/questions/4056083/how-to-address-the-numerical-instability-of-a-function-involving-erf-and-an-expo ?

Reply:
That is what I thought, but it still doesn't match. These are all basic ops and match with pytorch, but when we combine them, it doesn't have enough precision..

Reply:
Do you mean when you use a more numerically stable implementation from this link the results still do not match?

Reply:
Ah, I thought the link was about torch implementation.